docs/plugins/jenkins-github-integ.zh.md

```diff
@@ -133,8 +133,8 @@ tools:
         # release name of the jenkins helm chart, mandatory
         releaseName: 
       # GitHub repo where to put the pipeline script and project. mandatory
-      githubRepoUrl: https://github.com/aFlyBird0/jenkins-file-test
+      githubRepoUrl: https://github.com/xxx/xxx
```

Comment:
Maybe it is better to use some placeholder like `YOUR_GITHUB_ACCOUNT/YOUR_TEST_PROJECT_NAME` here.

Reply:
True, but this tends to trigger bad chain checks. Maybe we could add a `. *YOUR.*` link ignore rule?

Reply:
> True, but this tends to trigger bad links checks. Maybe we could add a `. *YOUR.*` link ignore rule?

Oh, I got it. Let me merge this pr first. And I think it's ok to add `.*YOUR_.*` ignore rule. @aFlyBird0